### PR TITLE
feat: details

### PR DESCRIPTION
**Pull request by remy**
Plus offline support.

Not ready for merge yet.

### CODE REVIEW

File: src/details/index.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+Large print schedules are available on request at the registration desk.
+
+We regret that there is no lift and consequently the Balcony and Bar can only be accessed by ascending a staircase of 25 steps.
+
+Left Logic would like to ensure that all delegates are able to have access therefore should you have any special requests please contact us in advance so we can accommodate your needs.
+
+## 2020!
+
+ffconf 2020 will take place on Friday 6th November! A limited number of early bird tickets will be released on Friday 8th November 2019 so get in early to guarantee your space (refunds will be issued until September 2020).
```
Comment (juliesharp) [inline src/details/index.md]: Probably 13th November

File: src/details/index.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+## Contact
+
+*   Email: [events@leftlogic.com](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/4osvh7uz/bWFpbHRvOmV2ZW50c0BsZWZ0bG9naWMuY29t&sa=D&ust=1572442324176000), or directly: [remy@leftlogic.com](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/wea2h6u9/bWFpbHRvOnJlbXlAbGVmdGxvZ2ljLmNvbQ%3D%3D&sa=D&ust=1572442324177000)& [julie@leftlogic.com](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/rqb8h9u9/bWFpbHRvOmp1bGllQGxlZnRsb2dpYy5jb20%3D&sa=D&ust=1572442324177000)
+*   Twitter & DM (we have open DMs): [@ffconf](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/0ghwhgu0/aHR0cHM6Ly90d2l0dGVyLmNvbS9mZmNvbmY%3D&sa=D&ust=1572442324178000)
+*   SMS or Phone: (+44) 7546 025 070 (this number will only be active during the event)
```
Comment (juliesharp) [inline src/details/index.md]: 07575351325

File: src/details/index.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+
+## Sponsors
+
+Please also take a few minutes to check out our sponsors and the links to their sites, without whom we couldn't run this event. Click on their links and check them all out. Seriously. We can't do this entire event without them.
+
+### Our Platinum sponsor
+
+*   [Mozilla](https://www.google.com/url?q=http://www.mozilla.com&sa=D&ust=1572442324180000)
```
Comment (juliesharp) [inline src/details/index.md]: strip google from urls

File: src/details/index.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+## Registration
+
+Doors open at 9am for registration and the conference starts at 9:40am sharp! There are no tickets for the conference, but you must bring a form of identification (bank card, driving licence etc) to pick up your delegate programme which also acts as your pass. Please keep your lanyard with you at all times as you will need it to get back into the venue.
+
+The day will finish by 6pm. The schedule will be in your programme and online, and if your device supports Service Workers, the site will also work offline.
+
+Note that by attending any of our events - workshops & conference, you are agreeing to our code of conduct: [https://2019.ffconf.org/code-of-conduct/](https://www.google.com/url?q=https://2019.ffconf.org/code-of-conduct/&sa=D&ust=1572442324158000).
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns or questions, please contact a member of staff (wearing ffconf t-shirts). Or contact Julie directly on (+44) 7546 025 070 by SMS or by calling.
```
Comment (juliesharp) [inline src/details/index.md]: 07575351325

File: src/details/index.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+## Reduce, reuse, recycle
+
+The coffee cups we use are compostable but please do bring your own travel mug and water bottle too.
+
+## Tweets, blog posts, etc
+
+Yes to all. Hashtag all the things with #ffconf.
+
+Feel free to follow the Twitter account [@ffconf](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/59bvh8up/aHR0cHM6Ly90d2l0dGVyLmNvbS9mZmNvbmY%3D&sa=D&ust=1572442324162000) for the latest info on the event.
```
Comment (juliesharp) [inline src/details/index.md]: strip urls

File: src/details/index.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+
+## Technology
+
+If you plan to bring a laptop/device, make sure it's fully charged for the day, as we only have a handful of charging stations in the venue.
+
+There is no wifi. I mean, it's a cinema, why would it have the internets! Network coverage however is very good, so you can tether your mobile phone or bring your own mifi or dongle, etc.
+
+## Post conference social
+
+We hope you will join us at OhSo Social again (and fingers crossed there won’t be a storm again this year!) There will be food available to order until 9PM and you collect your drinks voucher on the door. We will also have a range of board games so try an old favourite or learn a new game and make new friends.
```
Comment (juliesharp) [inline src/details/index.md]: ...collect your drinks voucher on the door so please bring your name badge/lanyard with you.

File: src/details/index.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+
+Oh So Social - 250a Kings Rd, Brighton BN1 1NB - on the beach, near the green doughnut
+
+Directions: [https://2019.ffconf.org/#locations](https://www.google.com/url?q=https://2019.ffconf.org/%23locations&sa=D&ust=1572442324166000)
+
+## Pre conference
+
+### A guide to Brighton's Best Wifi Cafes
+
+If you're spending some extra time in Brighton, it might be useful to drop by [a café with](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/gxinhwu7/aHR0cHM6Ly93d3cubWFjbWFuLmNvLnVrL2Jlc3QtY2FmZXMtaW4tYnJpZ2h0b24v&sa=D&ust=1572442324168000) [wifi](https://www.google.com/url?q=https://el2.convertkit-mail4.com/c/preview/gxinhwu7/aHR0cHM6Ly93d3cubWFjbWFuLmNvLnVrL2Jlc3QtY2FmZXMtaW4tYnJpZ2h0b24v&sa=D&ust=1572442324168000).
```
Comment (juliesharp) [inline src/details/index.md]: update with coffee map?

File: src/details/index.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+The ground floor is accessible to customers with limited mobility including wheelchair users.
+
+The Box Office, Kiosk, access toilet and screen are all located on the ground floor. From the foyer there is a small ramp from which to access the Kiosk and Screen. There is a wheelchair-accessible adapted toilet on the ground floor.
+
+Large print schedules are available on request at the registration desk.
+
+We regret that there is no lift and consequently the Balcony and Bar can only be accessed by ascending a staircase of 25 steps.
+
+Left Logic would like to ensure that all delegates are able to have access therefore should you have any special requests please contact us in advance so we can accommodate your needs.
```
Comment (juliesharp) [inline src/details/index.md]: not left logic but 'Remy and Julie want to ensure that everyone is able to have access ...

File: src/details/index.md (hunk 4 above)
Comment (juliesharp) [inline src/details/index.md]: DMs are also open

File: src/details/index.md (hunk 6 above)
Comment (remy) [reply src/details/index.md]: done

File: src/details/index.md (hunk 2 above)
Comment (remy) [reply src/details/index.md]: done

File: src/details/index.md (hunk 4 above)
Comment (remy) [reply src/details/index.md]: done

File: src/details/index.md (hunk 5 above)
Comment (remy) [reply src/details/index.md]: done

File: src/details/index.md (hunk 8 above)
Comment (remy) [reply src/details/index.md]: Done

File: src/details/index.md (hunk 3 above)
Comment (remy) [reply src/details/index.md]: done

File: src/details/index.md (hunk 1 above)
Comment (remy) [reply src/details/index.md]: Have changed, do we wait for confirmation?

**Pull request merged**